app/editor/src/features/admin/ingests/IngestList.tsx

```diff
@@ -42,15 +39,31 @@ export const IngestList: React.FC<IIngestListProps> = (props) => {
           onClick={() => navigate('/admin/ingests/0')}
         />
       </Row>
-      <GridTable
-        columns={columns}
-        isLoading={!!requests.length}
-        sorting={{ sortBy: [{ id: 'id', desc: false }] }}
-        paging={{ pageSizeOptions: { fromLocalStorage: true } }}
+      <IngestFilter
+        setGlobalFilter={(filter) => {
+          if (filter && filter.length) {
+            const value = filter.toLocaleLowerCase();
+            setItems(
+              ingests.filter(
```

Comment:
Simple filter implementation.  I need to create a generic way to do this and apply it to the FlexboxTable at some point.